### PR TITLE
L2 775 launchpad style adjustment

### PR DESCRIPTION
**Pull request by mstrasinskis**
# Motivation

Update launchpad/sns styles

https://user-images.githubusercontent.com/98811342/176726328-46c1abe0-e8bf-4094-9474-c73a4313e482.mp4

# Changes

- make sns `<main>` centered and wider
- header shadow
- z-index variables
- Card withArrow prop

# Tests

- Card withArrow

### CODE REVIEW

**Review by lmuntaner — APPROVED** [review]
LGTM

File: frontend/svelte/src/lib/components/ui/Card.svelte [hunk 1]
```diff
@@ -1,9 +1,13 @@
 <script lang="ts">
+  import IconArrowRight from "../../icons/IconArrowRight.svelte";
+
   export let role: "link" | "button" | "checkbox" | undefined = undefined;
   export let ariaLabel: string | undefined = undefined;
   export let selected: boolean = false;
   export let disabled: boolean | undefined = undefined;
   export let testId: string = "card";
+  export let style: "default" | "blue" = "default";
```
Comment (peterpeterparker) [inline frontend/svelte/src/lib/components/ui/Card.svelte:9]: - is `style` a reserved keyword?

- maybe another name than `blue`? who knows maybe in the future "blue" won't be "blue" anymore but "orange" or else

File: frontend/svelte/src/lib/components/ui/MainContentWrapper.svelte [hunk 2]
```diff
@@ -14,8 +14,9 @@
     overflow-x: hidden;
 
     &.sns {
+      margin: 0 auto;
       padding: var(--padding-4x) var(--padding-2x);
-      max-width: 1300px;
+      max-width: 1450px;
```
Comment (peterpeterparker) [inline frontend/svelte/src/lib/components/ui/MainContentWrapper.svelte:19]: can we have a global variable for this?

File: frontend/svelte/src/lib/themes/themes/dark.scss [hunk 3]
```diff
@@ -6,6 +6,8 @@
   --menu-box-shadow: var(--box-shadow);
   --interaction-box-shadow: 0px 4px 4px rgba(0, 0, 0, 0.25),
     1px 1px 2px rgba(0, 0, 0, 0.3);
+  --header-box-shadow: -1px -1px 2px rgba(255, 255, 255, 0.8),
+    1px 1px 2px rgba(0, 0, 0, 0.3);
```
Comment (peterpeterparker) [inline frontend/svelte/src/lib/themes/themes/dark.scss:10]: that's (`1px 1px 2px rgba(0, 0, 0, 0.3);`) the same of above value, should we have a property to not repeat the value multiple times?

File: frontend/svelte/src/lib/themes/variables.scss [hunk 4]
```diff
@@ -23,6 +23,11 @@
   --border-radius-5x: calc(5 * var(--border-radius));
 
   --z-index: 1;
+  --menu-z-index: calc(var(--z-index) + 996);
```
Comment (peterpeterparker) [inline frontend/svelte/src/lib/themes/variables.scss:26]: cool idea 👍

File: frontend/svelte/src/tests/mocks/sns-projects.mock.ts [hunk 5]
```diff
@@ -27,12 +27,12 @@ export const mockSnsSwapState = (rootCanisterId: Principal): SnsSwapState =>
     },
     [principal(1).toText()]: {
       rootCanisterId: principal(1),
-      myCommitment: undefined,
+      myCommitment: BigInt(0 * 100000000),
```
Comment (peterpeterparker) [inline frontend/svelte/src/tests/mocks/sns-projects.mock.ts:30]: so `BigInt(0)`?

File: frontend/svelte/src/lib/components/ui/MainContentWrapper.svelte (hunk 2 above)
Comment (mstrasinskis) [reply frontend/svelte/src/lib/components/ui/MainContentWrapper.svelte]: Good point! Done.

File: frontend/svelte/src/lib/components/ui/Card.svelte (hunk 1 above)
Comment (mstrasinskis) [reply frontend/svelte/src/lib/components/ui/Card.svelte]: My bad. Fixed.

File: frontend/svelte/src/lib/themes/themes/dark.scss (hunk 3 above)
Comment (mstrasinskis) [reply frontend/svelte/src/lib/themes/themes/dark.scss:10]: [A task](https://dfinity.atlassian.net/browse/L2-785) was created to not forget about it

**Review by peterpeterparker — APPROVED** [review]
LGTM, thanks

**Pull request merged**